[DRAFT] Remove www subdomain

schoork opened the pull request:
### What github issue is this PR for, if any?
N/A

### What changed, and why?
Redirects www.casavolunteertracking.org to casavolunteertracking.org.

<img width="1412" alt="Screenshot 2024-03-22 at 6 14 07 AM" src="https://github.com/rubyforgood/casa/assets/50247514/dee2a2a9-c6e5-4124-828f-cffa5fecd832">
<img width="1411" alt="Screenshot 2024-03-22 at 6 13 57 AM" src="https://github.com/rubyforgood/casa/assets/50247514/74874f6f-40ff-4d74-a2f6-928a345eef7f">


### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪


### Screenshots please :)


### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`![alt text](https://media.giphy.com/media/1nP7ThJFes5pgXKUNf/giphy.gif)`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

dependabot[bot] committed:
build(deps-dev): bump bullet from 7.1.4 to 7.1.6

Bumps [bullet](https://github.com/flyerhzm/bullet) from 7.1.4 to 7.1.6.
- [Changelog](https://github.com/flyerhzm/bullet/blob/main/CHANGELOG.md)
- [Commits](https://github.com/flyerhzm/bullet/compare/7.1.4...7.1.6)

---
updated-dependencies:
- dependency-name: bullet
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

sarvaiyanidhi committed:
Display Volunteer name next to Case Number

sarvaiyanidhi committed:
Removed Volunteer name displayed next to case number from table view

Natblow committed:
update labels on the form

Co-Authored-By: Josh Smith <jmsmith1018@gmail.com>

sarvaiyanidhi committed:
Code changes to show volunteer name in case number droplist

sarvaiyanidhi committed:
Fixed Spec issue to get current_user

sarvaiyanidhi committed:
Fix ERB lint issue

sarvaiyanidhi committed:
Merge branch 'main' of github.com:sarvaiyanidhi/casa into display-assigned-volunteer

Natblow committed:
wip

elasticspoon committed:
feat: adds models

wop

elasticspoon committed:
feat: adds model seeding

elasticspoon committed:
feat: adds controllers

elasticspoon committed:
feat: adds views

JoshDevHub committed:
Style other expenses section in one row

Co-authored-by: Nate <natblow@hotmail.fr>

JoshDevHub committed:
Style back button

Co-authored-by: Natblow <natblow@hotmail.fr>

JoshDevHub committed:
Style optional text in form headings

Co-authored-by: Natblow <natblow@hotmail.fr>

elasticspoon committed:
feat: frontend changes based on feedback

elasticspoon committed:
fix: issues from review

fix: chevron respect select status

chore: removes a TODO

refactor: modal view component into subcomponents
adds tests

refactor: kebab to generic dropdown menu

fix: unsafe save

fix: view changes

dependabot[bot] committed:
build(deps): bump chart.js from 4.4.1 to 4.4.2

Bumps [chart.js](https://github.com/chartjs/Chart.js) from 4.4.1 to 4.4.2.
- [Release notes](https://github.com/chartjs/Chart.js/releases)
- [Commits](https://github.com/chartjs/Chart.js/compare/v4.4.1...v4.4.2)

---
updated-dependencies:
- dependency-name: chart.js
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

elasticspoon committed:
fix: docker timeout

elasticspoon committed:
fix: removes scoping

After talking with PMs alphabetical scoping doesn't make sense.

Topics should be set in the order they are on organization, maybe
in the future they should even be swappable in order.

Natblow committed:
style add-other-expenses button

Co-authored-by: Josh Smith <jmsmith1018@gmail.com>

Natblow committed:
add dollar-sign to expenses input field

Co-authored-by: Josh Smith <jmsmith1018@gmail.com>

elasticspoon committed:
fix: code review comments

fix: order answers based on topics

fix: set flaky test pending

fix: remove TODOs

fix: lint

Commit:
5549 Edit Org causes a decrypt error

Commit:
removed super flaky tests

compwron committed:
Merge pull request #5565 from rubyforgood/5549-edit-org-decrypt-error

Edit Org causes a decrypt error

compwron committed:
improve factory_bot linter, fix flaky test, fix invalid linter

compwron committed:
Fix a factory

compwron committed:
lintr

tmorton committed:
Fix User#no_recent_sign_in to avoid pulling in unrelated users

JoshDevHub committed:
Additional expenses use new form styling

Co-authored-by: Natblow <natblow@hotmail.fr>

schoork committed:
Merge pull request #5517 from elasticspoon/5500-add-contact-topic-backend

dependabot[bot] committed:
build(deps): bump follow-redirects from 1.15.4 to 1.15.6

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.15.4 to 1.15.6.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.15.4...v1.15.6)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump strong_migrations from 1.6.4 to 1.8.0

Bumps [strong_migrations](https://github.com/ankane/strong_migrations) from 1.6.4 to 1.8.0.
- [Changelog](https://github.com/ankane/strong_migrations/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ankane/strong_migrations/compare/v1.6.4...v1.8.0)

---
updated-dependencies:
- dependency-name: strong_migrations
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

jp524 committed:
Modify volunteers#index system test to not use sleep

jp524 committed:
Skip flaky tests in Volunteers#index system tests

compwron committed:
Merge pull request #5487 from rubyforgood/dependabot/bundler/bullet-7.1.6

build(deps-dev): bump bullet from 7.1.4 to 7.1.6

jdsoteldo committed:
refactor script if statement

jdsoteldo committed:
extracted conditional into its own method

jdsoteldo committed:
added not found hash into process_casa_case_date method

JoshDevHub committed:
Merge branch 'main' into Update-"4.-Enter-Travel-Details"-section-on-Case-Contact-Form

elasticspoon committed:
feat: rework court report generation

Add generate docx modal with date pickers to filter dates of case
contacts to include in the generated report.

Modify current tests to open the modal before
attempting to click within it.

Allow range filtering of contacts in case report

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-env from 7.23.9 to 7.24.3

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.23.9 to 7.24.3.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.24.3/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump rdoc from 6.6.2 to 6.6.3.1

Bumps [rdoc](https://github.com/ruby/rdoc) from 6.6.2 to 6.6.3.1.
- [Release notes](https://github.com/ruby/rdoc/releases)
- [Changelog](https://github.com/ruby/rdoc/blob/master/History.rdoc)
- [Commits](https://github.com/ruby/rdoc/compare/v6.6.2...v6.6.3.1)

---
updated-dependencies:
- dependency-name: rdoc
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
Remove flaky tests

schoork committed:
Merge pull request #5575 from rubyforgood/5548-refactor-test-to-not-need-sleep

compwron committed:
Merge pull request #5571 from tmorton/fix-no-recent-sign-in

 Bug: Supervisor Weekly Mailer Lists Too Many Inactive Volunteers

compwron committed:
Merge pull request #5586 from rubyforgood/dependabot/bundler/rdoc-6.6.3.1

build(deps): bump rdoc from 6.6.2 to 6.6.3.1

compwron committed:
Merge pull request #5584 from rubyforgood/dependabot/npm_and_yarn/babel/preset-env-7.24.3

build(deps-dev): bump @babel/preset-env from 7.23.9 to 7.24.3

compwron committed:
Merge pull request #5574 from rubyforgood/dependabot/bundler/strong_migrations-1.8.0

build(deps): bump strong_migrations from 1.6.4 to 1.8.0

compwron committed:
Merge pull request #5572 from rubyforgood/dependabot/npm_and_yarn/follow-redirects-1.15.6

build(deps): bump follow-redirects from 1.15.4 to 1.15.6

dependabot[bot] committed:
build(deps): bump rails from 7.1.3 to 7.1.3.2

Bumps [rails](https://github.com/rails/rails) from 7.1.3 to 7.1.3.2.
- [Release notes](https://github.com/rails/rails/releases)
- [Commits](https://github.com/rails/rails/compare/v7.1.3...v7.1.3.2)

---
updated-dependencies:
- dependency-name: rails
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

compwron committed:
Merge pull request #5559 from rubyforgood/dependabot/npm_and_yarn/chart.js-4.4.2

build(deps): bump chart.js from 4.4.1 to 4.4.2

dependabot[bot] committed:
build(deps): bump bootstrap from 5.3.2 to 5.3.3

Bumps [bootstrap](https://github.com/twbs/bootstrap) from 5.3.2 to 5.3.3.
- [Release notes](https://github.com/twbs/bootstrap/releases)
- [Commits](https://github.com/twbs/bootstrap/compare/v5.3.2...v5.3.3)

---
updated-dependencies:
- dependency-name: bootstrap
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump bootstrap-scss from 5.3.2 to 5.3.3

Bumps [bootstrap-scss](https://github.com/crissdev/bootstrap-scss) from 5.3.2 to 5.3.3.
- [Release notes](https://github.com/crissdev/bootstrap-scss/releases)
- [Commits](https://github.com/crissdev/bootstrap-scss/compare/v5.3.2...v5.3.3)

---
updated-dependencies:
- dependency-name: bootstrap-scss
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

jp524 committed:
Modify system tests for LearningHours#index to verify different behaviour for Admin/Supervisors

elasticspoon committed:
refactor: expose court report methods

Expose methods in court report context to make unit
testing easier.

Write additional unit tests.
Remove old tests.

Remove redundant tests, some behavior is
already tested by the Dates object.

sarvaiyanidhi committed:
Fix to include contact_topic_answer as well in chart value

sarvaiyanidhi committed:
Lint fix

JoshDevHub committed:
Correct position of Pundit check in expenses form

JoshDevHub committed:
Correct input group for when expenses already exist

compwron committed:
Merge pull request #5592 from sarvaiyanidhi/ns_fix_case_contact_creation_graph

Fix for Case Contact chart for Case Contact with notes calculation

compwron committed:
test that i wish would fail but does not yet

elasticspoon committed:
fix: improved query

dependabot[bot] committed:
build(deps): bump @babel/core from 7.24.0 to 7.24.4

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.24.0 to 7.24.4.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.24.4/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

elasticspoon committed:
feat: use most recent court date in modal

On the case show page it should autofill the start date of
the date range with the most recent court date.

This most closely mirrors existing behavior.

elasticspoon committed:
fix: prevent seeding extra topic answers

Previously every single case contact would get seeded with
answers to every single topic. They would only be seeded with
answers to topics that belong to the contacts' organization.

elasticspoon committed:
fix: lint and forgot about test

compwron committed:
Merge pull request #5602 from elasticspoon/fix-topic-answer-seeding

fix: prevent seeding extra topic answers

compwron committed:
Merge pull request #5590 from elasticspoon/refactor-context

refactor: expose court report methods

dependabot[bot] committed:
build(deps): bump esbuild from 0.20.1 to 0.20.2

Bumps [esbuild](https://github.com/evanw/esbuild) from 0.20.1 to 0.20.2.
- [Release notes](https://github.com/evanw/esbuild/releases)
- [Changelog](https://github.com/evanw/esbuild/blob/main/CHANGELOG.md)
- [Commits](https://github.com/evanw/esbuild/compare/v0.20.1...v0.20.2)

---
updated-dependencies:
- dependency-name: esbuild
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

compwron committed:
Merge pull request #5534 from sarvaiyanidhi/display-assigned-volunteer

Display Volunteer name next to Case Number

compwron committed:
Merge pull request #5553 from rubyforgood/dependabot/npm_and_yarn/bootstrap-5.3.3

build(deps): bump bootstrap from 5.3.2 to 5.3.3

compwron committed:
Merge pull request #5555 from rubyforgood/dependabot/bundler/rails-7.1.3.2

build(deps): bump rails from 7.1.3 to 7.1.3.2

compwron committed:
Merge pull request #5566 from rubyforgood/dependabot/npm_and_yarn/bootstrap-scss-5.3.3

build(deps): bump bootstrap-scss from 5.3.2 to 5.3.3

compwron committed:
spec improvements for case contacts

compwron committed:
Merge pull request #5570 from rubyforgood/lint

improve invalid factorybot linter, some specs

compwron committed:
PR test to see what fails

compwron committed:
Merge pull request #5577 from jdsoteldo/jd-refactor-import-case-date-script

Jd refactor import case date script

compwron committed:
standardrb --fix

compwron committed:
Merge pull request #5610 from rubyforgood/fix-lint-after-5577

standardrb --fix

compwron committed:
Merge pull request #5600 from rubyforgood/dependabot/npm_and_yarn/babel/core-7.24.4

build(deps): bump @babel/core from 7.24.0 to 7.24.4

compwron committed:
Merge pull request #5578 from JoshDevHub/Update-"4.-Enter-Travel-Details"-section-on-Case-Contact-Form

Update Travel details section on case contact form

compwron reviewed (changes requested):
this is a big strange diff, rebase please

compwron committed:
Merge pull request #5588 from rubyforgood/5513-write-system-test-for-supervisors-and-admins-seeing-learning-hours

Modify system tests for LearningHours#index to verify different behaviours for Admins/Supervisors vs Volunteers

compwron committed:
Merge pull request #5595 from rubyforgood/dependabot/npm_and_yarn/esbuild-0.20.2

build(deps): bump esbuild from 0.20.1 to 0.20.2

compwron committed:
Merge pull request #5593 from elasticspoon/filtered-reports-generation

Filtered reports generation

compwron committed:
Merge pull request #5598 from rubyforgood/issue-5597-supervisor-weekly-email-wrong-org-volunteers-bug

extra test for supplier mailer weekly inactive volunteers section

dependabot[bot] committed:
build(deps): bump trix from 2.0.10 to 2.1.0

Bumps [trix](https://github.com/basecamp/trix) from 2.0.10 to 2.1.0.
- [Release notes](https://github.com/basecamp/trix/releases)
- [Commits](https://github.com/basecamp/trix/compare/v2.0.10...v2.1.0)

---
updated-dependencies:
- dependency-name: trix
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @fortawesome/fontawesome-free from 6.5.1 to 6.5.2

Bumps [@fortawesome/fontawesome-free](https://github.com/FortAwesome/Font-Awesome) from 6.5.1 to 6.5.2.
- [Release notes](https://github.com/FortAwesome/Font-Awesome/releases)
- [Changelog](https://github.com/FortAwesome/Font-Awesome/blob/6.x/CHANGELOG.md)
- [Commits](https://github.com/FortAwesome/Font-Awesome/compare/6.5.1...6.5.2)

---
updated-dependencies:
- dependency-name: "@fortawesome/fontawesome-free"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump sass from 1.71.1 to 1.75.0

Bumps [sass](https://github.com/sass/dart-sass) from 1.71.1 to 1.75.0.
- [Release notes](https://github.com/sass/dart-sass/releases)
- [Changelog](https://github.com/sass/dart-sass/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sass/dart-sass/compare/1.71.1...1.75.0)

---
updated-dependencies:
- dependency-name: sass
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-env from 7.24.3 to 7.24.4

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.24.3 to 7.24.4.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.24.4/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump amazing_print from 1.5.0 to 1.6.0

Bumps [amazing_print](https://github.com/amazing-print/amazing_print) from 1.5.0 to 1.6.0.
- [Release notes](https://github.com/amazing-print/amazing_print/releases)
- [Changelog](https://github.com/amazing-print/amazing_print/blob/master/CHANGELOG.md)
- [Commits](https://github.com/amazing-print/amazing_print/compare/v1.5.0...v1.6.0)

---
updated-dependencies:
- dependency-name: amazing_print
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump net-smtp from 0.4.0.1 to 0.5.0

Bumps [net-smtp](https://github.com/ruby/net-smtp) from 0.4.0.1 to 0.5.0.
- [Release notes](https://github.com/ruby/net-smtp/releases)
- [Changelog](https://github.com/ruby/net-smtp/blob/master/NEWS.md)
- [Commits](https://github.com/ruby/net-smtp/compare/v0.4.0.1...v0.5.0)

---
updated-dependencies:
- dependency-name: net-smtp
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump faker from 3.2.3 to 3.3.1

Bumps [faker](https://github.com/faker-ruby/faker) from 3.2.3 to 3.3.1.
- [Release notes](https://github.com/faker-ruby/faker/releases)
- [Changelog](https://github.com/faker-ruby/faker/blob/main/CHANGELOG.md)
- [Commits](https://github.com/faker-ruby/faker/compare/v3.2.3...v3.3.1)

---
updated-dependencies:
- dependency-name: faker
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump spring from 4.1.3 to 4.2.0

Bumps [spring](https://github.com/rails/spring) from 4.1.3 to 4.2.0.
- [Release notes](https://github.com/rails/spring/releases)
- [Changelog](https://github.com/rails/spring/blob/main/CHANGELOG.md)
- [Commits](https://github.com/rails/spring/compare/v4.1.3...v4.2.0)

---
updated-dependencies:
- dependency-name: spring
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump shoulda-matchers from 6.1.0 to 6.2.0

Bumps [shoulda-matchers](https://github.com/thoughtbot/shoulda-matchers) from 6.1.0 to 6.2.0.
- [Release notes](https://github.com/thoughtbot/shoulda-matchers/releases)
- [Changelog](https://github.com/thoughtbot/shoulda-matchers/blob/main/CHANGELOG.md)
- [Commits](https://github.com/thoughtbot/shoulda-matchers/compare/v6.1.0...v6.2.0)

---
updated-dependencies:
- dependency-name: shoulda-matchers
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

elasticspoon committed:
fix: remove sleeps

Some system specs were relying on sleep
to wait for client side JS to finish.

This changes the behavior to be more reliable by using
built in Capybara methods.

Also removes animations in systems tests.

elasticspoon committed:
fix: court report document show

Apparently in controllers relating to court reports,
when were never scoping correctly. If there were
multiple reports with the same case number it would
show the incorrect report.

This is possible if two orgs have cases with the same number.

compwron committed:
Merge pull request #5614 from rubyforgood/dependabot/npm_and_yarn/trix-2.1.0

build(deps): bump trix from 2.0.10 to 2.1.0

compwron committed:
Merge pull request #5615 from rubyforgood/dependabot/npm_and_yarn/fortawesome/fontawesome-free-6.5.2

build(deps): bump @fortawesome/fontawesome-free from 6.5.1 to 6.5.2

compwron committed:
Merge pull request #5625 from elasticspoon/fix-case-show-spec

fix: select correct report in controller

compwron committed:
Merge pull request #5616 from rubyforgood/dependabot/npm_and_yarn/sass-1.75.0

build(deps): bump sass from 1.71.1 to 1.75.0

compwron committed:
Merge pull request #5617 from rubyforgood/dependabot/npm_and_yarn/babel/preset-env-7.24.4

build(deps-dev): bump @babel/preset-env from 7.24.3 to 7.24.4

compwron committed:
Merge pull request #5618 from rubyforgood/dependabot/bundler/amazing_print-1.6.0

build(deps): bump amazing_print from 1.5.0 to 1.6.0

compwron committed:
Merge pull request #5619 from rubyforgood/dependabot/bundler/net-smtp-0.5.0

build(deps): bump net-smtp from 0.4.0.1 to 0.5.0

compwron committed:
Merge pull request #5620 from rubyforgood/dependabot/bundler/faker-3.3.1

build(deps): bump faker from 3.2.3 to 3.3.1

compwron committed:
Merge pull request #5622 from rubyforgood/dependabot/bundler/shoulda-matchers-6.2.0

build(deps-dev): bump shoulda-matchers from 6.1.0 to 6.2.0

compwron committed:
Merge pull request #5623 from elasticspoon/fix-remove-sleeps

fix: remove sleeps

compwron committed:
Merge pull request #5621 from rubyforgood/dependabot/bundler/spring-4.2.0

build(deps-dev): bump spring from 4.1.3 to 4.2.0
